src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -57,10 +57,11 @@ foreach ($defaultEnvs as $envName => $envValue) {
     }
 }
 
+$isWindows = '\\' === DIRECTORY_SEPARATOR;
```

Comment:
please revet this change: it's unrelated and it slows down reviewing

Reply:
👌